### DISCUSSION TITLE
Does scenicplus support analysis on single-cell sequencing data alone?

### DISCUSSION THREAD

**Discussion by aaronwolf1995** [aertslab/scenicplus#170] [Q&A]
If I have only the single-cell transcriptomics data, is there any chance I could enjoy the use of scenicplus? Or could I use sc-ATAC-seq data with the similar experiment design available online?
Thanks for your help and suggestions!

**Comment by SidG13** [comment]
You should use [SCENIC](https://github.com/aertslab/SCENIC) for scRNAseq data. It's a very useful tool.
